src/pages/billing/List.js

```diff
@@ -39,27 +40,37 @@ const styles = {
   }
 }
 
-const ProjectListItem = ({ project: { projectName, roles }, isActive }) => {
-  return div({ role: 'listitem' }, [
-    _.includes(billingRoles.owner, roles) && h(Clickable, {
-      style: {
-        ...styles.projectListItem(isActive),
-        color: isActive ? colors.dark() : colors.accent()
-      },
-      href: `${Nav.getLink('billing')}?${qs.stringify({
-        selectedName: projectName,
-        type: 'project'
-      })}`,
-      onClick: () => {
-        Ajax().Metrics.captureEvent(Events.billingProjectOpenFromList, {
-          billingProjectName: projectName
-        })
-      },
-      hover: Style.navList.itemHover(isActive),
-      'aria-current': isActive ? 'location' : false
-    }, [projectName])
+const ProjectListItem = (() => {
```

Comment:
I think you still need the props defined here? I am a bit surprised it didn't complain that they were removed.

Reply:
It's defined in the closure that's returned.